### PR TITLE
Initial updates to auth.js & authManager.js

### PR DESCRIPTION
**Pull request by vishalshrm539**

### CODE REVIEW

**Review by JEAxero — COMMENTED** [review]
Hi. Can you describe what these changes do (and if there we should see any UI changes with the addition of the iframe)? And will these changes also be made to the Angular SDK and Web Components SDK soon?

**Comment by vishalshrm539** [comment]
> Hi. Can you describe what these changes do (and if there we should see any UI changes with the addition of the iframe)? And will these changes also be made to the Angular SDK and Web Components SDK soon?

Yes John, similar changes will be done in other SDKs as well. I've just created this PR now for Vinod to review once, will add the details and refine the changes as per his suggestions.

**Comment by JEAxero** [comment]
Thanks. I just want to make sure that I know what to expect and that we're keeping these in sync across SDKs!

**Comment by vishalshrm539** [comment]
If `iframeLoginUI` property within sdk-config.json is set to 'true` then we will be able to login using an iframe instead of a popup window for all the re-authentications i.e. all authentications except for the first one.

File: src/helpers/authManager.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ const initOAuth = (bInit) => {
         : `${window.location.origin}${window.location.pathname}`,
     authService: sdkConfigAuth.authService,
     appAlias: sdkConfigAuth.appAlias || '',
-    useLocking: true
+    useLocking: true,
```
Comment (vrseraphin) [inline src/helpers/authManager.js:114]: Don't think this line required modification.  Not sure why trailing comma was added (perhaps no harm)? Not sure how lint rules deal with this whether they suggest you add that or not add that.

Comment (vishalshrm539) [reply src/helpers/authManager.js]: Removed this comma.

**Review by vrseraphin — APPROVED** [review]
Looks good!  Thanks for earlier requested changes.

**Pull request merged**